src/foam/dashboard/view/Metric.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dashboard.view',
+  name: 'Metric',
+  extends: 'foam.u2.Element',
+
+  imports: [
+    'data',
+    'visualizationWidth',
+    'visualizationHeight'
+  ],
+
+  css: `
+    ^ {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      padding: 20px;
+      background: $backgroundDefault;
```

Comment:
Use semantic tokens

Reply:
done, but would like another round of review to be sure I got it right.